Userland/Applications/VideoPlayer/VideoPlayerWidget.cpp

```diff
@@ -349,6 +362,14 @@ void VideoPlayerWidget::set_seek_mode(Video::PlaybackManager::SeekMode seek_mode
     m_use_fast_seeking->set_checked(seek_mode == Video::PlaybackManager::SeekMode::Fast);
 }
 
+void VideoPlayerWidget::set_sizing_mode(VideoSizingMode sizing_mode)
```

Comment:
Should this follow the `if (m_value != new_value) return;` pattern we have elsewhere (esp in LibGUI) so that we don't cause unecessary churn with the config server if you set the sizing mode to what it already is?